frontend/public/components/default-resource.jsx

```diff
@@ -17,30 +18,65 @@ import {
 const { common } = Cog.factory;
 const menuActions = [...common];
 
-const Header = props => <ListHeader>
-  <ColHead {...props} className="col-xs-6 col-sm-4" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-xs-6 col-sm-4" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-sm-4 hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
-</ListHeader>;
+const computeColumnSizes = (columns, isResourceNamespaced) => {
```

Comment:
I'd say we want a maximum number of columns to show at each screen width breakpoint and drop columns with lower priority.

Alternately, we could make the table horizontally scrollable.

Reply:
Hmm.. so both options make sense to have.

>...we want a maximum number of columns to show at each screen width...

thats 12 columns for all the widths. Not sure if we want to render more then 6 columns (mainly on narrow screens) cause in case of more data the output table could get confusing.

![1](https://user-images.githubusercontent.com/1668218/47348576-0e76f580-d6b2-11e8-858c-af47bf917b26.png)

> Alternately, we could make the table horizontally scrollable.

I like the idea, since the output would be more `oc` consistent. Could we have both ? With a `Show full table` link as a switcher?